src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -14679,15 +14641,15 @@ public class LibraryClientTest {
   @SuppressWarnings("all")
   public void listStringsTest2() {
     String nextPageToken = "";
-    ResourceName stringsElement = ArchiveBookName.of("[ARCHIVE]", "[BOOK]");
+    ResourceName stringsElement = ArchivedBookName.of("[ARCHIVE]", "[BOOK]");
     List<ResourceName> strings = Arrays.asList(stringsElement);
     ListStringsResponse expectedResponse = ListStringsResponse.newBuilder()
       .setNextPageToken(nextPageToken)
       .addAllStrings(UntypedResourceName.toStringList(strings))
       .build();
     mockLibraryService.addResponse(expectedResponse);
 
-    ResourceName name = ArchiveBookName.of("[ARCHIVE]", "[BOOK]");
+    ResourceName name = ArchivedBookName.of("[ARCHIVE]", "[BOOK]");
```

Comment:
Putting this comment here to note our offline the discussions result:
`ArchiveBookName` was changed to `ArchivedBookName` as a result of fixing basically broken configuration used for the tests